apertiumPairs.json

```diff
@@ -1,243 +1,106 @@
 ,
-	{"stems": 586, "repo": "incubator", "created": "2011-07-28", "last_updated": "2012-04-19", "lg2": "sme", "lg1": "eus", "coordinates1": [23.0, 69.0], "coordinates2": [-2.5, 43.3], "direction": ">"}
+	{"stems": 586, "repo": "incubator", "created": "2011-07-28", "last_updated": "2012-04-19", "lg2": "sme", "lg1": "eus", "direction": ">"}
 ,
-	{"stems": 250, "repo": "incubator", "created": "2014-06-15", "last_updated": "2014-09-02", "lg2": "hbs", "lg1": "fin", "coordinates1": [19.0, 45.5], "coordinates2": [25.0, 62.0], "direction": "<>"}
+	{"stems": 250, "repo": "incubator", "created": "2014-06-15", "last_updated": "2014-09-02", "lg2": "hbs", "lg1": "fin", "direction": "<>"}
 ,
-	{"stems": 1000, "repo": "incubator", "created": "2014-05-25", "last_updated": "2014-06-09", "lg2": "isl", "lg1": "fin", "coordinates1": [-19.0, 64.0], "coordinates2": [25.0, 62.0], "direction": "<>"}
+	{"stems": 1000, "repo": "incubator", "created": "2014-05-25", "last_updated": "2014-06-09", "lg2": "isl", "lg1": "fin", "direction": "<>"}
 ,
-	{"stems": 281, "repo": "incubator", "created": "2014-09-10", "last_updated": "2014-09-11", "lg2": "ita", "lg1": "fin", "coordinates1": [13.0, 42.5], "coordinates2": [25.0, 62.0], "direction": "<>"}
+	{"stems": 281, "repo": "incubator", "created": "2014-09-10", "last_updated": "2014-09-11", "lg2": "ita", "lg1": "fin", "direction": "<>"}
 ,
-	{"stems": 283, "repo": "incubator", "created": "2014-08-21", "last_updated": "2014-08-22", "lg2": "spa", "lg1": "fin", "coordinates1": [-3.0, 41.0], "coordinates2": [25.0, 62.0], "direction": "<>"}
+	{"stems": 283, "repo": "incubator", "created": "2014-08-21", "last_updated": "2014-08-22", "lg2": "spa", "lg1": "fin", "direction": "<>"}
 ,
-	{"stems": 1893, "repo": "incubator", "created": "2014-04-27", "last_updated": "2014-06-09", "lg2": "swe", "lg1": "fin", "coordinates1": [15.0, 59.0], "coordinates2": [25.0, 62.0], "direction": "<>"}
+	{"stems": 1893, "repo": "incubator", "created": "2014-04-27", "last_updated": "2014-06-09", "lg2": "swe", "lg1": "fin", "direction": "<>"}
 ,
-	{"stems": 194, "repo": "incubator", "created": "2013-05-11", "last_updated": "2014-06-09", "lg2": "eng", "lg1": "fra", "coordinates1": [-1.0, 53.0], "coordinates2": [2.5, 47.0], "direction": "<>"}
+	{"stems": 194, "repo": "incubator", "created": "2013-05-11", "last_updated": "2014-06-09", "lg2": "eng", "lg1": "fra", "direction": "<>"}
 ,
-	{"stems": 52, "repo": "incubator", "created": "2014-02-09", "last_updated": "2014-06-09", "lg2": "rus", "lg1": "isl", "coordinates1": [39.0, 57.0], "coordinates2": [-19.0, 64.0], "direction": "<>"}
+	{"stems": 52, "repo": "incubator", "created": "2014-02-09", "last_updated": "2014-06-09", "lg2": "rus", "lg1": "isl", "direction": "<>"}
 ,
-	{"stems": 1, "repo": "incubator", "created": "?", "last_updated": "2014-01-07", "lg2": "ita", "lg1": "ita", "coordinates1": [13.0, 42.5], "coordinates2": [13.0, 42.5], "direction": "?"}
```

Comment:
@jonorthwash 

Weird, seems like there is a path going from Italy back to itself. Should I just remove this?